frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/PackageOptionsModal/PackageOptionsModal.tsx

```diff
@@ -13,39 +16,58 @@ interface IPackageOptionsModal {
   preInstallQuery?: string;
   postInstallScript?: string;
   onExit: () => void;
+  installType?: InstallType;
+  labels?: ILabelIdentifier[];
+  labelsIncludeAny?: boolean;
 }
 
 const PackageOptionsModal = ({
   installScript,
   preInstallQuery,
   postInstallScript,
   onExit,
+  installType,
+  labels,
+  labelsIncludeAny,
 }: IPackageOptionsModal) => {
   return (
     <Modal className={baseClass} title="Options" onExit={onExit}>
       <>
-        <p>
+        <p className={`${baseClass}__header`}>
           Options are read-only. To change options, delete software and add
           again.
         </p>
-        <div className={`${baseClass}__form-inputs`}>
-          <Editor
-            readOnly
-            wrapEnabled
-            maxLines={10}
-            name="install-script"
-            value={installScript}
-            helpText="Fleet will run this command on hosts to install software."
-            label="Install script"
-            labelTooltip="For security agents, add the script provided by the vendor."
-          />
-          {preInstallQuery && (
-            <div className={`${baseClass}__input-field`}>
-              <span>Pre-install condition:</span>
+        <div className="form">
+          <>
+            {installType && (
+              <div className="form-field">
+                <div className="form-field__label">Install</div>
+                {installType[0].toUpperCase() + installType.slice(1)}
+              </div>
+            )}
+            {!!labels?.length && (
+              <div className="form-field">
+                <div className="form-field__label">Target</div>
+                <div className="fleet-labels-list-header">
+                  Software will only be available for install on hosts that{" "}
+                  {!labelsIncludeAny && "don't "}have <b>any</b> of these
+                  labels:
+                </div>
+                <div className={`${baseClass}__fleet-labels-list`}>
```

Comment:
Nit: Consideration to use a list instead of `<div>`s

```
 <ul className={`${baseClass}__fleet-labels-list`}>
      {labels.map((label) => (
        <li key={label.id} className={`${baseClass}__fleet-label`}>
          {label.name}
        </li>
      ))}
    </ul>
```